### PR TITLE
Fix test errors

### PR DESCRIPTION
**Pull request by shaazk**
### What does this PR do?
- Fix test errors
### Description of Task to be completed?
- Add test timeout
### How should this be manually tested?
- You have to clone the repo git clone https://github.com/Stackup-Rwanda/fan7-bn-backend.git
- Then After the repo cloned cd project-folder and run npm I to install all dependencies
- Run the app by npm test
Any background context you want to provide?
N/A

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,8 +59,8 @@
   },
   "scripts": {
     "pre-test": "cross-env NODE_ENV=test npm run undo:db && cross-env NODE_ENV=test npm run migrate:db && cross-env NODE_ENV=test npm run seed:db",
-    "test": "npm run pre-test && cross-env NODE_ENV=test nyc --reporter=text --reporter=html mocha --timeout=5000 --require @babel/polyfill --require @babel/register --exit ./src/test/index.test.js",
-    "stop-linux": "killall node",
+    "test": "npm run pre-test && cross-env NODE_ENV=test nyc --reporter=text --reporter=html mocha --timeout=50000 --require @babel/polyfill --require @babel/register --exit ./src/test/index.test.js",
```
Comment (ajimae) [inline package.json]: Don't you think 50 seconds is too much? 🤔

**Review by ajimae — COMMENTED** [review]
Hi @shaazk I left you a comment please do well to check it out.
🥂

File: package.json (hunk 1 above)
Comment (shaazk) [reply package.json]: Yes, its actually alot, i reduced it to 10000

Comment (shaazk) [reply package.json]: we are still getting a timeout problem. I have moved it up to 20000

**Review by ajimae — APPROVED** [review]
LGTM

**Pull request merged**